assets/scss/components/vendor/slick/_slick.scss

```diff
@@ -45,7 +45,7 @@
 }
 
 .slick-next {
-    right: -10px;
+    right: -5px;
```

Comment:
Could we use variable here?

Reply:
As you can see media values are quite specific - on this aspect ratio we still have width of mobile version but no enough place for content because of small height. So I decided to remove text for such cases. It can affect devices like iPhone 4s

Reply:
👍